src/resolvers/get-engine-hp.js

```diff
@@ -0,0 +1,19 @@
+const getEngineHorsepowerFromDescription = description => {
+  const match = description.match(/(\d{2}|\d{3})(| )hk|(\d{2}|\d{3})(| )hp/)
```

Comment:
Perhaps this should somewhat match: https://github.com/connectedcars/node-integration/blob/7dc557cec1b344ffda395ba872eaef57146e6735/src/semler/autocore.js#L198
I don't think we have any cases of `HP` yet. If you wanna keep hp, add a test case for it.

Reply:
Added tests and fixed usage of a wrong match group (2 should've been 3)